ecosystem/indexer-grpc/indexer-grpc-data-service/README.md

```diff
@@ -15,19 +15,36 @@ Indexer GRPC data service fetches data from both cache and file store.
 ```yaml
 health_check_port: 8083
 server_config:
-    data_service_grpc_listen_address: 0.0.0.0:50052
     whitelisted_auth_tokens: 
       - "token1"
       - "token2"
     file_store_config:
       file_store_type: GcsFileStore
       gcs_file_store_bucket_name: indexer-grpc-file-store-bucketname
     data_service_grpc_tls_config:
+      data_service_grpc_listen_address: 0.0.0.0:50055
       cert_path: /path/to/cert.cert
       key_path: /path/to/key.pem
+    data_service_grpc_non_tls_config:
+      data_service_grpc_listen_address: 0.0.0.0:50057
     redis_read_replica_address: 127.0.0.1:6379
 ```
 
+### Config Explanation
+
+* `whitelisted_auth_tokens`: list of tokens that are allowed to consume the endpoint.
+* `file_store_config`: GCS to archive the transactions
+* `data_service_grpc_tls_config`: TLS endpoint exposed
+* `data_service_grpc_non_tls_config`: Non-TLS endpoint exposed
```

Comment:
add some comments on what the difference is and why we have both?

Reply:
done.